tests/test_authenticated_reddit.py

```diff
@@ -96,6 +96,13 @@ def test_submission_hide_and_unhide_batch(self):
         new = list(sub.get_new(limit=5, params={'show': 'all', 'count': 3}))
         self.assertTrue(not any(item.hidden for item in new))
 
+        submissions = list(self.r.get_subreddit('all').get_new(limit=300))
+        fullnames = [submission.fullname for submission in submissions]
+        self.r.hide(fullnames)
```

Comment:
Can we make the group functionality a separate test to ensure the original cassette still works as expected?